client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -61,6 +61,7 @@ const startTask = ( dispatch, task, siteId ) => {
 
 const skipTask = ( dispatch, task, siteId ) => {
 	dispatch( requestSiteChecklistTaskUpdate( siteId, task.id ) );
+	dispatch( requestHomeLayout( siteId ) );
```

Comment:
There's likely a similar race condition here but we can fix/bulletproof in a follow up